tools/query_breakdown/src/main/java/com/google/bigquery/Main.java

```diff
@@ -17,19 +21,24 @@ public static void main(String[] args) {
       System.out.println("You have not specified the input file");
```

Comment:
When this happens, shouldn't you need to throw an exception to stop the execution?